_posts/2020-09-26-bring-your-own-datatypes.md

```diff
@@ -265,18 +267,19 @@ We hope this will encourage datatype researchers
   we hope this will spark interest
   in custom datatypes
   within the deep learning community.
-The Bring Your Own Datatypes framework
-  partially exists in TVM at the moment,
-  and more will be merged in (including full documentation)
-  in the coming months.
+For more documentation about the Bring Your Own Datatypes framework
+  please visit the developer tutorial [Bring Your Own Datatypes to TVM](https://tvm.apache.org/docs/tutorials/dev/bring_your_own_datatypes.html#sphx-glr-tutorials-dev-bring-your-own-datatypes-py){:target='_blank'}.
```

Comment:
What do you think of,
"...please visit the [Bring Your Own Datatypes to TVM] developer tutorial"?
I think it flows a little better...or even just
"...please visit the [developer tutorial]"?

Reply:
yep agreed! thanks!